utils/retrieve-state/src/main.rs

```diff
@@ -6,6 +6,7 @@ use std::{
     str::FromStr,
 };
 
+use casper_types::EraId;
```

Comment:
Import in the wrong place

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2531/commits/9a4261fae4aa6c6834052ff12514d12d47c9e858